src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -7597,5 +7597,36 @@ struct B
 }",
                 MainDescription("struct B"));
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public async Task TestNormalFuncSynthesizedLambdaType()
+        {
+            await TestAsync(
+@"class C
+{
+    void M()
+    {
+        $$var v = (int i) => i.ToString();
+    }
+}",
+                MainDescription("delegate TResult System.Func<in T, out TResult>(T arg)"),
```

Comment:
Shouldn't this just be `System.Func<...>`? I don't understand the combination of `System.Func...` with `delegate` and a method signature.

Reply:
So Func is a delegate, so we show that (just like we say if something is a class/struct). And for delegates we show the signature as that's what's relevant a lot of the time.  So this is standard for how we've shown this information for about 15 years :-)